aspnetcore/docfx.json

```diff
@@ -46,7 +46,7 @@
       "ms.topic": "conceptual",
       "feedback_system": "GitHub",
       "feedback_github_repo": "aspnet/docs",
-      "feedback_product_url": "https://developercommunity.visualstudio.com/spaces/61/index.html" 
+      "feedback_product_url": "https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md" 
```

Comment:
you'll need to configure this differently based on the feedback from Dan.

Reply:
`@DamianEdwards` trumps @danroth27 
@danroth27  do you want your feedback of https://github.com/aspnet/home/issues or your bosses?